src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -605,52 +616,55 @@ def get_available_view_names(self, rom_name: str):
+            msg += f"\n Available TBROM names are: {self.tbrom_names}"
             self._raise_error(msg)
 
         view_names = list(self._tbrom_info[rom_name][self.TBROM_VIEWS_KEY])
 
         if len(view_names) == 0:
-            msg = f"No views are available for given rom_name: {rom_name}."
+            msg = f"No views are available for the given RON name: {rom_name}."
             self._log_message(msg, level=PyTwinLogLevel.PYTWIN_LOG_WARNING)
 
         return view_names
 
     def get_image_filepath(self, rom_name: str, view_name: str, evaluation_time: float = 0.0):
         """
-        Get the image file associated to a Reduced Order Model (ROM) available in the TwinModel and evaluated at the
-        given time instant. The image file shows the field results of the ROM in the given predefined view.
+        Get the image file associated with a ROM (reduced order model) in the twin model and evaluated at a
+        given time instant.
+
+        The image file shows the field results of the ROM in the given predefined view.
 
         Parameters
         ----------
         rom_name : str
-            This is the name of a ROM model that is available in the TwinModel. See TwinModel.tbrom_names property to
-            get a list of available ROM model.
+            Name of the ROM model. To get a list of available ROMs, see
+            the :attr:`pytwin.TwinModel.tbrom_names' attribute.
         view_name : str
-            The view name associated to the rendering view with which the ROM results are displayed. See
-            TwinModel.get_available_view_names method to get a list of available rendering view names for a given ROM.
-        evaluation_time: float
-            This is the evaluation time at which you want to get the snapshot file. The method returns None if no
-            snapshot file is available at this evaluation_time. Two evaluation times can be distinguished up to 6 digits
-            after the comma.
+            View name associated with the rendering view in which ROM results are displayed. To get
+            a list of available rendering view names for a given ROM, use the
+            :func:`pytwin.TwinModel.get_available_view_names` method.
+        evaluation_time: float, optional
+            Evaluation time at which to get the snapshot file. The default is ``0.0``. If no
+            snapshot file is available at the time specified, this method returns ``None``.
+            Two evaluation times can be distinguished up to six digits after the comma.
 
         Raises
         ------
         TwinModelError:
-            It raises an error if TwinModel has not been initialized.
-            It raises an error if TwinModel does not include any TBROM.
-            It raises an error if rom_name is not available.
-            It raises an error if view_name is not available.
+            If ``TwinModel`` has not been initialized.
+            If ``TwinModel`` does not include any TBROM.
```

Comment:
As indicated earlier, use "If the ``TwinModel`` object..." or "If twin model object..."

Reply:
done